scripts/fix-nx-cache.mjs

```
@@ -0,0 +1,145 @@
+      cwd: path.join(__dirname, '..'),
+      stdio: 'inherit'
+    })
+    console.log('  ✓ jest-taro-helper:clean 执行成功')
+  } catch (error) {
+    console.error('  ✗ jest-taro-helper:clean 执行失败:', error.message)
+    throw error
+  }
+}
+
+async function getDbName() {
+  console.log('\n4. 获取 .db 文件名...')
```

ESLint

[error] Unexpected console statement.

(no-console)